### PR TITLE
Issue 110: Stats on product pages is invisible

### LINKED ISSUE #110: Bug: Stats on product pages is invisible
Screenshot 2023-06-26 at 4 41 13 PM

https://main--sgws-creative--hlxsites.hlx.page/bloominbrands/p/6

The stats should be visible below the logo-banner.

### PR DESCRIPTION
**Pull request by allentsliu**
## Issue

Fix #110 

## 🔗 Test URLs:

- Before: https://main--sgws-creative--hlxsites.hlx.page/bloominbrands/p/6
- After: https://issue-110--sgws-creative--hlxsites.hlx.page/bloominbrands/p/6

## 📝 Description:
Screen Shot 2023-06-27 at 2 07 46 PM

### CODE REVIEW

**Comment by tmathern** [comment]
In the authored page, switch the format from PNG for the hero/poster images to JPEG. THis will help lighthouse scores too.

File: scripts/scripts.js [hunk 1]
```diff
@@ -359,7 +360,8 @@ export function decorateSectionBackgrounds(main) {
     if (backgroundPicture) {
       section.classList.add('background-image');
       const pictureParent = backgroundPicture.closest('p.picture');
-      section.append(backgroundPicture);
+      const image = backgroundPicture.querySelector('img');
+      section.append(createOptimizedPicture(image.src, image.alt, true));
```
Comment (tmathern) [inline scripts/scripts.js:364]: You could actually attempt to set a width here, since it would be the width of the page.

**Review by tmathern — CHANGES_REQUESTED** [review]
I would attempt to:
- switch from PNG to JPEG for the image (there is no transparency)
- set a width/height to the image

File: scripts/scripts.js (hunk 1 above)
Comment (allentsliu) [reply scripts/scripts.js:364]: It seems like the createOptimizedPicture function will insert a width=750 by default `/bloominbrands/p/media_1294bb1e6fb32a045b99efb4d47357dca217f588a.jpeg?width=750&format=jpeg&optimize=medium` 
I have also tried switching from PNG to JPEG for the image, but that doesn't seem to help :(

**Comment by allentsliu** [comment]
@tmathern  could you approve this PR when you get a chance? Thanks!

**Review by tmathern — APPROVED** [review]

**Pull request merged**